deploy/olm-catalog/ibm-iam-operator/3.6.0/ibm-iam-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -232,6 +232,33 @@ metadata:
               }
             ]
           }
+        },
+        {
+          "apiVersion": "operator.ibm.com/v1alpha1",
+          "kind": "OperandBindInfo",
+          "metadata": {
+            "name": "ibm-iam-bindinfo"
+            "namespace": "ibm-common-services"
+          },
+          "spec": {
+            "operand": "ibm-iam-operator",
+            "registry": "common-service",
+            "description": "Binding information that should be accessible to iam adopters",
+            "bindings": [
```

Comment:
Bindings is not a list, please refer to this example https://github.com/IBM/ibm-licensing-operator/blob/master/deploy/olm-catalog/ibm-licensing-operator/1.1.0/ibm-licensing-operator.v1.1.0.clusterserviceversion.yaml#L31.

Please hold this PR today, let's wait for the decision if we need to bump the operator API version.

Reply:
@horis233 I have updated the PR. Sure, I will hold it until the decision is made regarding the api version.